stage_descriptions/producing-messages-04-ps7.md

```diff
@@ -0,0 +1,43 @@
+In this stage, you'll add support for successfully producing a single record.
+
+## Producing records
+
+When a Kafka broker receives a `Produce` request, it needs to validate that the topic and partition exist (using the `__cluster_metadata` topic's log file), store the record in the appropriate log file using Kafka's on-disk format, and return a successful response.
+
+The record must be persisted to the topic's log file at `<log-dir>/<topic-name>-<partition-index>/00000000000000000000.log` using Kafka's RecordBatch format.
+You can refer to the official Kafka docs for the [RecordBatch format](https://kafka.apache.org/documentation/#recordbatch).
```

Comment:
We don't need an extra sentence here. Just link to the docs above when we mention `RecordBatch`.